### DISCUSSION TITLE
Slingbox 500 Factory Reset

### DISCUSSION THREAD

**Discussion by DoogieHowitzerWMD** [GerryDazoo/Slinger#238] [Q&A]
I just got another Slingbox 500 on eBay, but I seem to be having trouble performing a factory reset on it.

I get two flashes after holding the reset button down for about 10 seconds or so, and the unit takes a minute or two to start up again. It asks for the connection setting (wired Ethernet) but has trouble connecting to the internet upon trying that connection. If I do a soft reset to try again, it connects with my router and starts up normally (as if I had programmed it). The firmware its running is the most current version, so it doesnt seem to be going back to factory settings.

If I keep holding the reset button down for over a minute, the unit just seems to repeatedly soft reset and not perform anything that seems like a hard reset. I dont remember having this lind of trouble when hard resetting my other boxes. Is there something I'm not doing, or did I maybe just get a bad slingbox?

**Comment by sp222222** [comment]
try using the mac address to force it to static IP. I got lucky with mine I bought on ebay last month as the mac sticker and the actual MAC address did not match and it threw me off for a bit until I realized that.  anyways. use wired ethernet force static IP and like someone else put in another thread make another sling email acct and use it to join/configure

**Reply by DoogieHowitzerWMD** [reply]
I had the static IP set for the MAC address on the slingbox and I could ping it. It just couldn't "connect to the internet" when trying to do the setup.

I tried opening a port that I assigned to MAC address/static ip, and tried turning UPnP on for it to grab a port in its own, and neither allowed it to "connect to the internet".

I suppose i could try connecting to it with the slinnger server, but I dont know if the old info eas wiped, as I didnt have to configure a choice of cable box or associate the Slingbox with an account.

**Comment by GerryDazoo** [comment]
I think if you try the setup assistant you'll see all  the video source configuration has been wiped out. When I was grabbing packet captures of the factory reset I saw the box automatically grabbing the latest firmware from the web.

**Reply by DoogieHowitzerWMD** [reply]
I wasnt able to get to the part where I could configure the video source or select my cable box manufacturer. It was stuck on the internet connection (but I could ping the device and see it from my router) It wouldnt connect in the setup stage, but soft resetting the box from the setup stage had the box bypass all that and go to the normal menu screen for a box that has been configured. I have no idea what account it's associated with at that point.

I suppose the account likely won't matter going forward, but setting up the choice of cable box would simplify having to configure the remote file for Slinger.

**Reply by Toei79** [reply]
![IMG-20221208-WA0003](https://user-images.githubusercontent.com/79808610/206486845-7ccbf477-21b5-47ed-9ba3-c2d2f49e03ef.jpg)
> I wasnt able to get to the part where I could configure the video source or select my cable box manufacturer. It was stuck on the internet connection (but I could ping the device and see it from my router) It wouldnt connect in the setup stage, but soft resetting the box from the setup stage had the box bypass all that and go to the normal menu screen for a box that has been configured. I have no idea what account it's associated with at that point.
> 
> I suppose the account likely won't matter going forward, but setting up the choice of cable box would simplify having to configure the remote file for Slinger.

i get one from ebay too one 500

then i did reset part, i get configuration screen internet connection, after it ask to upgrade fimware, what i supposed to do here?

i block some ports , like other threads mention,  first time i get the new firmware , but i dont update im afraid to get bricked?
so this time i block slingbox, and sling.com run this again and it cant update  , i cant pass this screen

then when i unplug and plug , slingbox dissappear from the router , the ip its gone

before that you see in the router this name
EchoStar Global B V 

instead slingbox 500 and after ip its gone from there.

**Comment by Mover480** [comment]
I don't think you need to block any ports, I didn't.  Not aware of any 500's being bricked.  After I did the factory reset, it made the internet connection and downloaded a firmware update.  After that, you can't select any cable service or remote; but it does work just fine.  You need an alternate method to control you STB.

**Reply by Toei79** [reply]
at the end i updated it , i ask on the slinger discord.

yea i get same thing , exactly i did , i update firmware then i get that screen, but i cant advance for same situation that you mention. then i run slinger, and its show me someone its using slingbox, so if i want keep setup or go to live, choose live and ''escape'' from the forever setup. now its normal. 

then curious why just some boxes they want get bricked , and others not? anyway i block same ports mention in other threads you never know.

what i see great you can even 10 at same time with slinger.  helps a lot those days we have fiber.

**Reply by GerryDazoo** [reply]
Have you tried to configure the remote with SlingPlayer desktop 5.0?

**Reply by Toei79** [reply]
I need check what version i have, it freezes in some part  of the setup, like connecting... idk its the word

**Reply by Toei79** [reply]
https://user-images.githubusercontent.com/79808610/206723114-d448df1b-ea6f-4648-abd3-53c9844f2c16.mp4

nothing look , idk what version i have . anyway stop there
